src/com/kimlew/javarenamefiles/RenameFiles.java

```diff
@@ -1,4 +1,4 @@
-package com.javarenamefiles;
+package com.kimlew.javarenamefiles;
```

Comment:
Ah, I see, changes from your other PR are in here too. OK, we'll go through the same stuff as last time.